shared/resources/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -113,12 +113,6 @@
       "idiom" : "ipad",
       "filename" : "appicon152x152.png",
       "scale" : "2x"
-    },
-    {
-      "size" : "83.5x83.5",
```

Comment:
We don't have those images, so there's an `unassigned children` warning generated.